conftest.py

```diff
@@ -47,7 +48,17 @@ def pytest_addoption(parser):
     parser.addoption(
         "--ci",
         action="store_true",
-        help="run just the tests appropiate for CI",
+        help="run just the tests appropriate for CI",
+    )
+    parser.addoption(
+        "--skips-file",
+        action="store",
+        help="file with tests to skip. Defaults to skips.txt"
+    )
+    parser.addoption(
+        "--xfails-file",
+        action="store",
+        help="file with tests to skip. Defaults to xfails.txt"
```

Comment:
I like defaulting the files, but wondering if we can default it here rather than below, e.g. specify a default like `pathlib.Path(__file__).parent / "skips.txt"`... kinda rusty on argparse things myself heh.